source/sonata_simulation.rst

```diff
@@ -227,6 +227,7 @@ A continuous injection of current.
    ============================== ========== ============ ==========================================
    amp_start                      float      Mandatory    The amount of current initially injected when the stimulus activates. Given in nA.
    amp_end                        float      Optional     If given, current is interpolated such that current reaches this value when the stimulus concludes. Otherwise, current stays at amp_start. Given in nA.
+   represents_physical_electrode  boolean    Optional     Default is False. If True, the signal will be implemented using a NEURON IClamp mechanism. The IClamp produce an electrode current which is not included in the calculation of extracellular signals, so this option should be used to represent a physical electrode. If the noise signal represents synaptic input, `represents_physical_electrode` should be set to False, in which case the signal will be implemented using a  MembraneCurrentSource mechanism, which is identical to IClamp, but produce a membrane current, which is included in the calculation of the extracellular signal. 
```

Comment:
If I'm understanding this right, the default value won't be a vanilla IClamp anymore? (what did it used to be in neurodamus @jorblancoa / @WeinaJi / @ferdonline ?).  Instead, `MembraneCurrentSource` is substituted, which uses `NONSPECIFIC_CURRENT i` instead of `ELECTRODE_CURRENT i`?

Will this impact how simulations are run, since the default is different?

Reply:
Yes, that is correct (the default was previously a vanilla IClamp). This will not have any impact on how simulations are run. It will have an impact on reporting, since now the stimulus current will be included in `i_membrane` instead of `IClamp`

Reply:
> It will have an impact on reporting, since now the stimulus current will be included in i_membrane instead of IClamp

I think we'll need to have broad confirmation that everyone who relies on reporting know about this then, since it's a change that it impacts them.  Probably @MWolfR, @romani79 should get their team to comment, @darshanmandge / @anilbey probably need to know, and likely others.

Reply:
We just discussed it with @joseph-tharayil. This change does not introduce any issues on bluecellulab side since the voltage traces produces will be identical.

Reply:
The changes does not affect any other single cell software. So, it is good from my side.

Reply:
Can we move forward with this and the PR in libsonata?
https://github.com/BlueBrain/libsonata/pull/354

cc: @mgeplf @joseph-tharayil

Reply:
It's been communicated to the circuits and connectomics teams, and no one's raised any objections yet, so I think we can move forward

Reply:
> It's been communicated to the circuits and connectomics teams,

Great! Can we get an ok on this ticket from them?
I just want to confirm that everyone is aware that we're moving away from a default that has been the standard for many years, to something different.